### PR TITLE
Unquarantine a pile of long-passing tests

### PR DESCRIPTION
**Pull request by analogrelay**
These are all tests that meet the following criteria:

1. Have passed 100% of runs for the past 30 days
1. Have had at least 150 total runs
1. Have been approved for unquarantining by the "owners" (Servers/SignalR team, etc.)

I've closed all the linked issues (for those that have issues). This may cause a little thrashing as some of these may pop back up again, but a reset seems in order here. For some tests, we've taken some specific actions to deflake them. For others, we believe that other infrastructure changes (moving to Helix, etc.) have improved reliability.

### CODE REVIEW

File: src/Servers/Kestrel/Kestrel/test/GeneratedCodeTests.cs
Comment (anurse) [inline src/Servers/Kestrel/Kestrel/test/GeneratedCodeTests.cs]: VS Code autoformatting. Sorry! Not going to take it out though, unless there's a lot of them :)

**Comment by anurse** [comment]
@msftbot make sure one of @Tratcher, @jkotalik or @halter73 approves before merging

**Comment by anurse** [comment]
@msftbot make sure one of @BrennanConroy or @halter73 approves before merging.

**Comment by anurse** [comment]
@msftbot make sure one of @Tratcher, @jkotalik or @halter73 approves before merging and do not merge unless @BrennanConroy approves

**Comment by anurse** [comment]
@msftbot please ensure at least 3 people sign off before merging.

**Review by halter73 — APPROVED** [review]

**Comment by anurse** [comment]
I couldn't quite get the condition I wanted :(.

**Review by BrennanConroy — APPROVED** [review]

**Review by Tratcher — APPROVED** [review]

**Comment by anurse** [comment]
Failure seems unrelated.

**Pull request merged**
